arrows/core/video_input_image_list.h

```diff
@@ -61,18 +65,19 @@ class KWIVER_ALGO_CORE_EXPORT video_input_image_list
                " in the \"image_reader\" config block."
                " Read an image list as a video stream." )
 
-  /// Constructor
   video_input_image_list();
   virtual ~video_input_image_list();
 
-  /// Get this algorithm's \link vital::config_block configuration block \endlink
-  virtual vital::config_block_sptr get_configuration() const;
+  /// Get this algorithm's
+  /// \link vital::config_block configuration block \endlink.
```

Comment:
I'm not 100% sure, but I thought these `///` comment lines could only be one line long.  We should verify that Doxygen is parsing this correctly when there are multiple lines.

Reply:
AFAICT that's only the case if a `///` is followed by a `/**` (a reason to not do that!). Otherwise, an explicit `@brief` is until the end of the paragraph, or a `JAVADOC_AUTOBRIEF` (ah, but I see we aren't using that) is until the full stop (`.`). Okay, so I'll add a `@brief`.

Reply:
Needs `@brief` / `\brief`. Fixed.

BTW, which one are we "supposed" to be using? Both are in use.